portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,15 +37,15 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  // Create Datastore instance to interact with the Datastore
+  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
private final

private because it doesn't need to be accessible from outside the class
final because it doesn't change after initialization here